protobuf/source_reflection_test.go

```diff
@@ -196,6 +196,44 @@ func TestReflectionRoutingHeaders(t *testing.T) {
 	}
 }
 
+func TestResolverAlreadyExists(t *testing.T) {
```

Comment:
Test creating a new resolver for the same service with a different list of peers